packages/fxa-settings/src/components/ButtonDownloadRecoveryKey/en.ftl

```diff
@@ -2,10 +2,11 @@
 ## Clicking on this button downloads a plain text file that contains the user's account recovery key
 ## The account recovery key can be used to recover data when users forget their account password
 
-# Button to download the account recovery key as a plain text file
+# Button to download the account recovery key as a plain text file and continue to the next step
+# "key" here refers to the "account recovery key"
 # .title will displayed as a tooltip on the button
-recovery-key-download-button-v2 = Download your account recovery key
-  .title = Download
+recovery-key-download-button-v3 = Download key & continue
```

Comment:
Should this be `&` or `&amp;`?

Reply:
@bcolsson Do you have a preference on `&` vs `and`?

Reply:
Either one works for localization (I've seen both used). That being said I don't know if I've seen it used much for FxA so in my very personal opinion I do like `and` more.

Reply:
I've brought up the question with UX/CX, I'm following their guidance for the copy 👍